examples/arithmetics/src/cli.ts

```diff
@@ -14,57 +14,61 @@ import { createArithmeticsServices } from './language-server/arithmetics-module'
-        const document = arithmeticsServices.documents.LangiumDocuments.getOrCreateDocument(URI.file(path.resolve(fileName)));
-        arithmeticsServices.documents.DocumentBuilder.build(document);
-        if (!document.parseResult) {
-            console.error(`Failed to parse the grammar file ${fileName}`);
-            process.exit(1);
+    const validationErrors = new DefaultDocumentValidator(arithmeticsServices)
+        .validateDocument(document)
+        .filter(e => e.severity === 1);
+    if (validationErrors.length > 0) {
+        console.error('There are validation errors:');
+        for (const validationError of validationErrors) {
+            console.error(colors.red(
+                `line ${validationError.range.start.line}: ${validationError.message} [${document.textDocument.getText(validationError.range)}]`
+            ));
         }
+        process.exit(1);
+    }
 
-        const validationErrors = new DefaultDocumentValidator(arithmeticsServices)
-            .validateDocument(document)
-            .filter(e => e.severity === 1);
-        if (validationErrors.length > 0) {
-            console.error('There are validation errors:');
-            for (const validationError of validationErrors) {
-                console.error(colors.red(
-                    `line ${validationError.range.start.line}: ${validationError.message} [${document.textDocument.getText(validationError.range)}]`
-                ));
+    const grammar = document.parseResult.value as Grammar;
+    if (isModule(grammar)) {
+        for (const [evaluation, value] of new ArithmeticsInterpreter().eval(grammar)) {
+            const cstNode = evaluation.expression.$cstNode;
+            if (cstNode) {
+                const line = document.textDocument.positionAt(cstNode.offset).line + 1;
+                console.log(`line ${line}:`, colors.green(cstNode.text), '===>', value);
             }
-            process.exit(1);
         }
+    }
+};
 
-        const grammar = document.parseResult.value as Grammar;
-        if (isModule(grammar)) {
-            for (const [evaluation, value] of new ArithmeticsInterpreter().eval(grammar)) {
-                const cstNode = evaluation.expression.$cstNode;
-                if (cstNode) {
-                    const line = document.textDocument.positionAt(cstNode.offset).line + 1;
-                    console.log(`line ${line}:`, colors.green(cstNode.text), '===>', value);
-                }
-            }
-        }
-    });
+export default function(): void{
+    const program = new Command();
+    program
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        .version(require('../package.json').version);
+
+    program
+        .command('eval')
+        .argument('<file>', 'the .calc file')
+        .description('calculate Evaluations in the .calc file')
+        .action(evalAction);
 
-program.parse(process.argv);
+    program.parse(process.argv);
+}
```

Comment:
@jbicker @Pluralia these changes are conflicting quite heavily with #246. In this case I think it would make sense if you talk to each other before we merge either of these PRs and make a plan how to resolve this.